.env.example

```diff
@@ -15,7 +15,8 @@ HELP_REQUISITES='{ "support_email": "help@doma.ai", "support_email_mobile": "hel
 
 # feature flags configuration
 FEATURE_FLAGS_CONFIG='["billing"]'
-FEATURE_TOGGLE_CONFIG='{"url": "http://localhost:3100/api/features", "apiKey": "key"}'
+#FEATURE_TOGGLE_CONFIG='{"url": "http://localhost:3100/api/features", "apiKey": "key"}'
```

Comment:
refactoring leftover?

Reply:
no, you can use url for features API or define it inside the env. It's a two examples

Reply:
Maybe to add additional service to docker-compose.yml to have GrowthBook out of the box?
It seems simpler in case when developer does not need to decide, whether to use GrowthBook or not.